react-app/src/containers/HomePage/HomePage.css

```diff
@@ -61,7 +61,7 @@ body {
     width: 35px;
     height: 35px;
 
-    background: url(icons8-male-user-96.png);
+    background: url(../Assets/Imagesicons8-male-user-96.png);
```

Comment:
Need to have a / between Images and image name

Reply:
Thanks for catching that, change made and pushing now